cli/command/environment.go

```diff
@@ -34,22 +35,17 @@ func (e *EnvironmentCommand) Command() cli.Command {
 						Usage: "type of the ec2 instances to use in the environment cluster",
 					},
 					cli.IntFlag{
-						Name:  "min-scale",
+						Name:  "scale",
 						Value: 0,
-						Usage: "minimum allowed scale of the environment cluster",
-					},
-					cli.IntFlag{
-						Name:  "max-scale",
-						Value: config.DefaultEnvironmentMaxScale,
-						Usage: "maximum allowed scale of the environment cluster",
+						Usage: "cluster size. setting scale explicitly, results in a static environment",
```

Comment:
I think this could be cleaned up a bit, maybe something like `specifies the number of instances in the cluster. Setting this value will create a static environment`

Reply:
✅